docs/mkdocs/docs/api/macros/nlohmann_json_serialize_enum.md

```diff
@@ -1,14 +1,16 @@
-# NLOHMANN_JSON_SERIALIZE_ENUM
+# NLOHMANN_JSON_SERIALIZE_ENUM and NLOHMANN_JSON_SERIALIZE_ENUM_STRICT
 
 ```cpp
 #define NLOHMANN_JSON_SERIALIZE_ENUM(type, conversion...)
+#define NLOHMANN_JSON_SERIALIZE_ENUM_STRICT(type, conversion...)
 ```
 
 By default, enum values are serialized to JSON as integers. In some cases this could result in undesired behavior. If an
 enum is modified or re-ordered after data has been serialized to JSON, the later de-serialized JSON data may be
 undefined or a different enum value than was originally intended.
 
-The `NLOHMANN_JSON_SERIALIZE_ENUM` allows to define a user-defined serialization for every enumerator.
+The `NLOHMANN_JSON_SERIALIZE_ENUM` and `NLOHMANN_JSON_SERIALIZE_ENUM_STRICT` macros allow to define a user-defined serialization for every enumerator.
```

Comment:
The wording of this sentence reads really weird to me, a possible alternative:
```suggestion
The `NLOHMANN_JSON_SERIALIZE_ENUM` and `NLOHMANN_JSON_SERIALIZE_ENUM_STRICT` macros allow specifying a user-defined serialization for every enumerator.
```
This then matches the language in the following sentence: "first value specified in the macro".

Reply:
much better, thank you